### DISCUSSION TITLE
Obtain edge attentions with Heterogeneous Graph Transformer?

### DISCUSSION THREAD

**Discussion by AdrianBZG** [pyg-team/pytorch_geometric#4684] [Q&A]
Hi, is it possible to obtain, given an input instance, the edge attentions when using a Heterogeneous Graph Transformer?

**Comment by rusty1s** [comment]
This is a good feature request, thanks for bringing it up. We would have to model it the same way as in other attention-based GNNs such as `GATConv` or `TransformerConv`: Store `alpha` during `message` computation [here](https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/nn/conv/hgt_conv.py#L185) and return it afterwards. Let me know if you have interest in contributing this feature!
